tests/utils/cluster/cluster.go

```diff
@@ -9,16 +9,24 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	testclient "github.com/test-network-function/cnfcert-tests-verification/tests/utils/client"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/nodes"
 )
 
 func IsClusterStable(clients *testclient.ClientSet) (bool, error) {
-	nodes, err := clients.Nodes().List(context.Background(), metav1.ListOptions{})
+	nodeInCluster, err := clients.Nodes().List(context.Background(), metav1.ListOptions{})
 	if err != nil {
 		return false, err
 	}
 
-	for _, node := range nodes.Items {
+	for _, node := range nodeInCluster.Items {
 		if node.Spec.Unschedulable {
+			fmt.Println("node", node.Name, "is cordoned, uncordoning it")
+			_, err := nodes.CordonOrUnCordonNodeByName(node.Name, clients, false)
```

Comment:
You don't need to do this, as it is just a personal preference: I'd create separate functions for each task: nodes.Cordon(name string) & node.Uncordon(name string). There's also no need to include neither the word "Node" nor "Name" in the function name, as it's obvious from the package "nodes" and the argument (name string).